src/lib.rs

```diff
@@ -37,15 +37,15 @@ mod range_map;
 mod mono_hash_map;
 mod stacked_borrows;
 
-use crate::fn_call::EvalContextExt as MissingFnsEvalContextExt;
-use crate::operator::EvalContextExt as OperatorEvalContextExt;
-use crate::intrinsic::EvalContextExt as IntrinsicEvalContextExt;
-use crate::tls::{EvalContextExt as TlsEvalContextExt, TlsData};
+pub use crate::fn_call::EvalContextExt as MissingFnsEvalContextExt;
+pub use crate::operator::EvalContextExt as OperatorEvalContextExt;
+pub use crate::intrinsic::EvalContextExt as IntrinsicEvalContextExt;
+pub use crate::tls::{EvalContextExt as TlsEvalContextExt, TlsData};
 use crate::range_map::RangeMap;
 #[allow(unused_imports)] // FIXME rustc bug https://github.com/rust-lang/rust/issues/53682
-use crate::helpers::{ScalarExt, EvalContextExt as HelpersEvalContextExt};
+pub use crate::helpers::{ScalarExt, EvalContextExt as HelpersEvalContextExt};
 use crate::mono_hash_map::MonoHashMap;
-use crate::stacked_borrows::{EvalContextExt as StackedBorEvalContextExt};
+pub crate::stacked_borrows::{EvalContextExt as StackedBorEvalContextExt};
```

Comment:
Uh... some of these, and certainly stacked borrows, have operations on the extension trait that are not supposed to be called anywhere outside their module.

The problem is that you cannot have a private method on an extension trait. But I don't care very strongly, as long as you don't expect any of these functions to remain stable, make any sense, or even keep having the same behavior. ;)

Reply:
I don't :laughing: I just made a swash public, even if I just needed a few, because I want to see if I can also render a few stacked borrow things.